packages/grafana-data/src/types/data.ts

```diff
@@ -21,6 +21,13 @@ export enum DataTopic {
   Annotations = 'annotations',
 }
 
+export enum LabelOperators {
+  Equals = '=',
+  NotEquals = '!=',
+  RegexMatch = '=~',
+  NotRegexMatch = '!=~',
```

Comment:
The non matching regex operator, `!=~`, I'm not certain on. According to the prometheus docs it should be `!~` but it may mean having to tweak the `labelRegexp`

https://prometheus.io/docs/prometheus/latest/querying/basics/#instant-vector-selectors